MapboxVision/Services/VideoSource/VideoSource.swift

```diff
@@ -5,6 +5,7 @@
 
 import Foundation
 import MapboxVisionCore
+import AVFoundation
```

Comment:
```suggestion
import CoreMedia
```
to avoid unneeded dependencies.